components/dashboard/src/feedback-form/FeedbackComponent.tsx

```diff
@@ -0,0 +1,160 @@
+    };
+
+    const emojiGroup = (width: number) => {
+        return (
+            <>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["starry"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img src={starryEyed} alt="starry eyed emoji" width={width || "24px"} title="starry" />
+                </button>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["happy"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img
+                        className="bottom-5 right-5 cursor-pointer"
+                        title="happy"
+                        src={happy}
+                        alt="happy emoji"
+                        width={width}
+                    />
+                </button>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["meh"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img
+                        className="bottom-5 right-5 cursor-pointer"
+                        title="meh"
+                        src={meh}
+                        alt="meh emoji"
+                        width={width}
+                    />
+                </button>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["crying"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img
+                        className="bottom-5 right-5 cursor-pointer"
+                        title="crying"
+                        src={crying}
+                        alt="crying emoji"
+                        width={width}
+                    />
+                </button>
+            </>
+        );
+    };
+    return (
+        <>
+            <h3 className="mb-4">Send Feedback</h3>
+            {selectedEmoji ? (
+                <>
+                    <div className="flex flex-col -mx-6 px-6 py-4 border-t border-b border-gray-200 dark:border-gray-800 bg-white dark:bg-gray-900">
+                        <div className="relative">
+                            <div className="absolute flex bottom-5 right-5">{emojiGroup(24)}</div>
```

Comment:
Do you know how I can reduce the spacing here? Can't figure it out 🙈

Reply:
> Do you know how I can reduce the spacing here?

Haven't checked, @laushinka! But feel free to leave this as is for now. Additionally, in case it helps, we can also drop the zoom-effect when the emojis are used within the text area.

Reply:
Done! Used `-space-x`